.travis.yml

```
@@ -6,30 +10,117 @@ branches:
     only:
         - master
 
-language: cpp
-compiler:
-    - g++
-    - clang
-
-sudo: false
-
-addons:
-    apt:
-        sources:
-            - ubuntu-toolchain-r-test
-            - george-edison55-precise-backports
-        packages:
-            - g++-4.9
-            - clang
-            - cmake-data
-            - cmake
+matrix:
+  include:
+    - name: "default build (C++14)"
+      compiler: g++-8
+      env:
+        - ARCH=amd64
+        #compiler
+        - COMPILER=g++-8
+        - ACXX=g++-8
+        - ACC=gcc-8
+        #build settings
+        - build_type=Release
+        - SANITIZER=false
+      addons:
+        apt:
+          packages: ['gcc-8', 'g++-8', 'cmake']
+
+    - name: "C++11 build"
+      compiler: g++-8
+      env:
+        - ARCH=amd64
+        #compiler
+        - COMPILER=g++-8
+        - ACXX=g++-8
+        - ACC=gcc-8
+        #build settings
+        - build_type=Release
```

let's build this with `Debug`.
```suggestion
        - build_type=Debug
```